pycqed/measurement/waveform_control/circuit_builder.py

```diff
@@ -854,7 +866,8 @@ def sweep_n_dim(self, sweep_points, body_block=None, body_block_func=None,
                         qb_names=all_ro_qubits, **final_kwargs)
 
                 segblock = self.sequential_blocks(
-                        'segblock', [prep, this_body_block, final, ro])
+                    'segblock', [prep, this_body_block, final, ro],
+                    disable_block_counter=True)
```

Comment:
Should this be hardcoded to True here in sweep_n_dim? In sequential_blocks it's default False. I am not sure where this feature is useful so it's hard for me to judge.

Reply:
The True is intended here, and actually the reason why I implemented the disable_block_counter feature in sequential_blocks. I have now added a comment that explains why the True is needed here.

Reply:
Thanks!